conf/sources-dist.json

```diff
@@ -10,10 +10,9 @@
   "alexa2": {
```

Comment:
If you ask me, this file change should be part of a separate PR

Reply:
Yes don't know why it went into it

Reply:
reverted